<html>
<head>
<title>TestNG:  TestSuite1</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>TestSuite1</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>8/6/0</td>
</tr><tr>
<td>Started on:</td><td>Sun Dec 02 16:57:08 ICT 2018</td>
</tr>
<tr><td>Total time:</td><td>159 seconds (159778 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='Railway.LoginTest.TC03()'><b>TC03</b><br>Test class: Railway.LoginTest<br>Test method: User cannot log into Railway with invalid password</td>
<td><div><pre>java.lang.AssertionError: expected [There was a problem with your login and/or errors exist in your form.] but found [Invalid username or password. Please try again.]
	at Railway.LoginTest.TC03(LoginTest.java:35)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1117871068", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1117871068'><pre>java.lang.AssertionError: expected [There was a problem with your login and/or errors exist in your form.] but found [Invalid username or password. Please try again.]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at Railway.LoginTest.TC03(LoginTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>1</td>
<td>Railway.LoginTest@f6c48ac</td></tr>
<tr>
<td title='Railway.LoginTest.TC05()'><b>TC05</b><br>Test class: Railway.LoginTest<br>Test method: System shows message when user enters wrong password several times</td>
<td><div><pre>java.lang.AssertionError: expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]
	at Railway.LoginTest.TC05(LoginTest.java:47)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace43650267", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace43650267'><pre>java.lang.AssertionError: expected [You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.] but found [Invalid username or password. Please try again.]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at Railway.LoginTest.TC05(LoginTest.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>4</td>
<td>Railway.LoginTest@f6c48ac</td></tr>
<tr>
<td title='Railway.ChangePasswordTest.TC09()'><b>TC09</b><br>Test class: Railway.ChangePasswordTest<br>Test method: User can change password</td>
<td><div><pre>java.lang.AssertionError: expected [Your password has been updated] but found [Your password has been updated!]
	at Railway.ChangePasswordTest.TC09(ChangePasswordTest.java:26)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace359806817", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace359806817'><pre>java.lang.AssertionError: expected [Your password has been updated] but found [Your password has been updated!]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at Railway.ChangePasswordTest.TC09(ChangePasswordTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>15</td>
<td>Railway.ChangePasswordTest@239963d8</td></tr>
<tr>
<td title='Railway.RegisterTest.TC11()'><b>TC11</b><br>Test class: Railway.RegisterTest<br>Test method: User can't create account while password and PID fields are empty</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [Invalid password length.] but found [Invalid password length],
	expected [Invalid ID length.] but found [Invalid ID length]
	at Railway.RegisterTest.TC11(RegisterTest.java:37)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1690050988", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1690050988'><pre>java.lang.AssertionError: The following asserts failed:
	expected [Invalid password length.] but found [Invalid password length],
	expected [Invalid ID length.] but found [Invalid ID length]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at Railway.RegisterTest.TC11(RegisterTest.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>Railway.RegisterTest@13deb50e</td></tr>
<tr>
<td title='Railway.ForgotPasswordTest.TC12()'><b>TC12</b><br>Test class: Railway.ForgotPasswordTest<br>Test method: Errors display when password reset token is blank</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [Password Change Form] but found [Password Reset Instructions Form],
	expected [The password reset token is incorrect or may be expired. Visit the forgot password page to generate a new one.] but found [Could not reset password. Please correct the errors and try again.],
	expected [The password reset token is invalid.] but found [Please enter your password reset token. It should have been sent to you in an email.]
	at Railway.ForgotPasswordTest.TC12(ForgotPasswordTest.java:29)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1842173497", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1842173497'><pre>java.lang.AssertionError: The following asserts failed:
	expected [Password Change Form] but found [Password Reset Instructions Form],
	expected [The password reset token is incorrect or may be expired. Visit the forgot password page to generate a new one.] but found [Could not reset password. Please correct the errors and try again.],
	expected [The password reset token is invalid.] but found [Please enter your password reset token. It should have been sent to you in an email.]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at Railway.ForgotPasswordTest.TC12(ForgotPasswordTest.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>22</td>
<td>Railway.ForgotPasswordTest@3abbfa04</td></tr>
<tr>
<td title='Railway.ForgotPasswordTest.TC13()'><b>TC13</b><br>Test class: Railway.ForgotPasswordTest<br>Test method: Errors display if password and confirm password don't match when resetting password</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [Password Change Form] but found [Password Reset Instructions Form],
	expected [The password reset token is incorrect or may be expired. Visit the forgot password page to generate a new one.] but found [Could not reset password. Please correct the errors and try again.],
	expected [The password reset token is invalid.] but found [Please enter your password reset token. It should have been sent to you in an email.],
	expected [Password Change Form] but found [Password Reset Instructions Form]
	at Railway.ForgotPasswordTest.TC13(ForgotPasswordTest.java:43)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1568159144", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1568159144'><pre>java.lang.AssertionError: The following asserts failed:
	expected [Password Change Form] but found [Password Reset Instructions Form],
	expected [The password reset token is incorrect or may be expired. Visit the forgot password page to generate a new one.] but found [Could not reset password. Please correct the errors and try again.],
	expected [The password reset token is invalid.] but found [Please enter your password reset token. It should have been sent to you in an email.],
	expected [Password Change Form] but found [Password Reset Instructions Form]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at Railway.ForgotPasswordTest.TC13(ForgotPasswordTest.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>25</td>
<td>Railway.ForgotPasswordTest@3abbfa04</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='Railway.LoginTest.TC01()'><b>TC01</b><br>Test class: Railway.LoginTest<br>Test method: User can log into Railway with valid username and password</td>
<td></td>
<td>9</td>
<td>Railway.LoginTest@f6c48ac</td></tr>
<tr>
<td title='Railway.LoginTest.TC02()'><b>TC02</b><br>Test class: Railway.LoginTest<br>Test method: User can't login with blank "Username" textbox</td>
<td></td>
<td>1</td>
<td>Railway.LoginTest@f6c48ac</td></tr>
<tr>
<td title='Railway.LoginTest.TC04()'><b>TC04</b><br>Test class: Railway.LoginTest<br>Test method: Login page displays when un-logged User clicks on "Book ticket" tab</td>
<td></td>
<td>1</td>
<td>Railway.LoginTest@f6c48ac</td></tr>
<tr>
<td title='Railway.LoginTest.TC06()'><b>TC06</b><br>Test class: Railway.LoginTest<br>Test method: Additional pages display once user logged in</td>
<td></td>
<td>5</td>
<td>Railway.LoginTest@f6c48ac</td></tr>
<tr>
<td title='Railway.RegisterTest.TC07()'><b>TC07</b><br>Test class: Railway.RegisterTest<br>Test method: User can create new account</td>
<td></td>
<td>5</td>
<td>Railway.RegisterTest@13deb50e</td></tr>
<tr>
<td title='Railway.LoginTest.TC08()'><b>TC08</b><br>Test class: Railway.LoginTest<br>Test method: User can't login with an account hasn't been activated</td>
<td></td>
<td>1</td>
<td>Railway.LoginTest@f6c48ac</td></tr>
<tr>
<td title='Railway.RegisterTest.TC10()'><b>TC10</b><br>Test class: Railway.RegisterTest<br>Test method: User can't create account with "Confirm password" is not the same with "Password"</td>
<td></td>
<td>1</td>
<td>Railway.RegisterTest@13deb50e</td></tr>
<tr>
<td title='Railway.BookTicketTest.TC14()'><b>TC14</b><br>Test class: Railway.BookTicketTest<br>Test method: User can book 1 ticket at a time</td>
<td></td>
<td>5</td>
<td>Railway.BookTicketTest@57fffcd7</td></tr>
</table><p>
</body>
</html>